zink/codegen/src/lib.rs

```diff
@@ -34,3 +37,9 @@ pub fn event(input: TokenStream) -> TokenStream {
     let input = parse_macro_input!(input as DeriveInput);
     event::parse(input)
 }
+
```

Comment:
please provide docs for this proc-macro, for example https://github.com/zink-lang/zink/blob/main/zink/codegen/src/lib.rs#L8, and the generated docs https://docs.zink-lang.org/rustdocs/zink_codegen/derive.Event.html